CMakeLists.txt

```diff
@@ -980,6 +980,12 @@ set(IREE_PUBLIC_INCLUDE_DIRS "${IREE_COMMON_INCLUDE_DIRS}"
 
 add_subdirectory(build_tools/benchmarks)
 
+#-------------------------------------------------------------------------------
+# IREE GitHub CI tools
+#-------------------------------------------------------------------------------
+
+add_subdirectory(build_tools/github_actions)
```

Comment:
Can you collapse all these `build_tools/*` includes into one section? The root `CMakeLists.txt` is already pretty large, and I don't think we gain much by splitting it up this way.

```
#-------------------------------------------------------------------------------
# IREE build tools
#-------------------------------------------------------------------------------

add_subdirectory(build_tools/benchmarks)
add_subdirectory(build_tools/github_actions)
add_subdirectory(build_tools/python)
```

(Or add a `CMakeLists.txt` to `build_tools/` and do the subdirectory includes there)

Reply:
Done